jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -2631,14 +2631,6 @@ public boolean isInterface() {
 public native boolean isPrimitive();
 
 /*[IF INLINE-TYPES]*/
-/**
- * Answers true if the receiver represents a primitive class type. Array classes
- * return false.
- *
- * @return	true if receiver is primitive class type, and false otherwise.
- */
-native boolean isPrimitiveClass();
```

Comment:
There are references of this method in JIT code like: https://github.com/eclipse-openj9/openj9/blob/6036c034f10ce416854c5182b861c7c38d431661/runtime/compiler/codegen/J9RecognizedMethodsEnum.hpp#L62

Reply:
Thanks. I removed the remaining references.